java-engine/lib/src/main/java/unleash/engine/Context.java

```diff
@@ -0,0 +1,69 @@
+package unleash.engine;
```

Comment:
We typically use the namespace `io.getunleash`, so this could/should probably be `io.getunleash.engine`

Reply:
Great suggestion, thanks! Fixed